### PR TITLE
add support for universal robot's e-series

### PR DESCRIPTION
**Pull request by AustinDeric**
apply changes from: 
https://github.com/ros-industrial/ur_modern_driver/pull/216

### CODE REVIEW

**Comment by AustinDeric** [comment]
@geoffreychiou please review and merge.

File: launch/ur10e_bringup.launch
Comment (geoffreychiou) [inline launch/ur10e_bringup.launch]: I remember not being able to run this launch file out of the box due to these args existing. Can you verify?

**Review by geoffreychiou — CHANGES_REQUESTED** [review]

**Comment by gomezc** [comment]
@AustinDeric @geoffreychiou 
Can we not do an PR to plusone-robotics:ur_moderner_driver? I'd like a separate branch as I believe this is not backwards compatible with the CB series controller; then we can have one branch for the CB and one branch for the E and just switch branches depending on which robot we are using for which "cell"/IPC.

**Comment by gomezc** [comment]
Also I'm ready to test once this gets approved. @AustinDeric can you make this a priority for Monday morning?

File: launch/ur10e_bringup.launch
Comment (AustinDeric) [reply launch/ur10e_bringup.launch]: fixed.

**Comment by shaun-edwards** [comment]
I believe it can be made backwards compatible.  Managing multiple branches introduces complexity. We should only do this if we have to.

**Comment by AustinDeric** [comment]
>I believe it can be made backwards compatible. Managing multiple branches introduces complexity. We should only do this if we have to.

The changes to update this driver to the new universal spec were identical to previous updates and follows the convention.  There is no reason to believe its not backward compatible, but we should test it to make sure.

**Review by geoffreychiou — APPROVED** [review]

**Comment by geoffreychiou** [comment]
Went ahead and approved the changes, i'll leave it to you guys to decide when to merge :smile:.

**Comment by AustinDeric** [comment]
ur_post_modern_driver tested on URSoftware 5.1.0.40195 and 3.5.4.10845. going to merge.

3.5.4.10845 was tested with an existing moveit package.  

5.1.0.40195 [didn't work when tested with existing moveit packages](https://github.com/ros-industrial/ur_modern_driver/pull/216#issuecomment-433993841) so i published a message to the /ur_driver/URScripts topic:

```
#!/usr/bin/env python
import rospy
import roslib
import std_msgs.msg


if __name__ == '__main__':

    rospy.init_node('test_URScript', anonymous = True)

    publisher = rospy.Publisher('/ur_driver/URScript', std_msgs.msg.String, queue_size=1)

    payload = "def driverProg():\n\
  movej([0.246289, -1.67353, 1.55276, -1.44812, -1.57027, -1.2908], t=0, r=0.05)\n\
  movej([0.246273, -1.67355, 1.55278, -1.4481, -1.57024, -1.29082], t=0.5, r=0.05)\n\
  movej([-0.553405, -1.80852, 1.61702, -1.46971, -1.68976, -2.12726], t=0.5, r=0.05)\n\
  movej([-0.652959, -1.67534, 2.22496, -2.16952, -1.75125, -0.666918], t=0.5, r=0)\n\
  movej([-0.676169, -1.58956, 2.29589, -2.33044, -1.75006, -0.690478], t=0.25, r=0)\n\
  stopj(3.14)\n\
  movej([-0.676218, -1.58964, 2.296, -2.33035, -1.75013, -0.690521], t=0, r=0.05)\n\
  movej([-0.652959, -1.67534, 2.22496, -2.16952, -1.75125, -0.666918], t=0.25, r=0.05)\n\
  movej([-0.553405, -1.80852, 1.61702, -1.46971, -1.68976, -2.12726], t=0.5, r=0.05)\n\
  movej([0.246273, -1.67355, 1.55278, -1.4481, -1.57024, -1.29082], t=0.5, r=0)\n\
  movej([0.246465, -1.64935, 1.90982, -1.82935, -1.57024, -1.29063], t=0.5, r=0)\n\
  stopj(3.14)\n\
end\n\
driverProg()"

    rospy.sleep(1)

    publisher.publish(payload)
    rospy.sleep(1)
```

**Comment by gomezc** [comment]
I affirm that robot motion looks good with new driver on CB series as well as E series. Merge away!

**Pull request merged**
